flink-connector-http/src/main/java/org/apache/flink/connector/http/table/lookup/HttpLookupConnectorOptions.java

```diff
@@ -137,6 +139,12 @@ public class HttpLookupConnectorOptions {
                             "Continue job on error. "
                                     + "This includes unsuccessful HTTP status codes and client side Exceptions, such as Connection Refused.");
 
+    public static final ConfigOption<String> LOGGING_LEVEL_FOR_HTTP =
+            ConfigOptions.key(HTTP_LOGGING_LEVEL)
+                    .stringType()
+                    .defaultValue(String.valueOf(HttpLoggingLevelType.MIN))
+                    .withDescription("Logging levels");
```

Comment:
Left a minimal description in line with the other options . I know it is not used.